README.md

```diff
@@ -44,6 +44,68 @@ module.exports = ruleComposer.joinReports([
 ]);
 ```
 
+You can access options set on a rule. The following example creates a modified version of the [`no-unused-expressions`](https://eslint.org/docs/rules/no-unused-expressions) rule which accepts a list of exceptions.
+
+```js
+
+/*
+  rule configuration:
+
+  {
+    "custom-no-unused-expressions": ["error", {
+      "whitelist": ["expect", "test"]
+    }]
+  }
+*/
+
+const ruleComposer = require('eslint-rule-composer');
+const eslint = require('eslint');
+const noUnusedExpressionsRule = new eslint.Linter().getRules().get('no-unused-expressions');
+
+module.exports = ruleComposer.filterReports(
+  noUnusedExpressionsRule,
+  (problem, metadata) => {
+    const firstToken = metadata.sourceCode.getFirstToken(problem.node);
+    const whitelist = metadata.options[0].whitelist;
+    return whitelist.includes(value) === false
+  }
+);
+```
+
+You can also access [shared settings](https://eslint.org/docs/user-guide/configuring#adding-shared-settings) from the current ESLint configuration file:
+
+
+```js
+
+/*
+  .eslint.json:
+
+  {
+    "settings": {
+      "tokenWhitelist": ["expect", "test"]
```

Comment:
I'm a bit confused about this example -- shouldn't `tokenWhitelist` be in the rule options rather than `settings`? It seems like the option is only applicable to this one rule.

Reply:
To clarify, I agree that it's useful to have access to settings -- I just think this isn't a good example of the utility because this type of configuration would be better in `options`.

It's probably not really necessary to have separate examples for `options` and `settings`, so it might be simplest to just remove this example.